tests/buildTools/src/test/scala/tests/GradleBuildToolSuite.scala

```diff
@@ -287,6 +287,28 @@ class GradleBuildToolSuite extends BaseBuildToolSuite {
     4
   )
 
+  List("8", "11").foreach { version =>
```

Comment:
Thanks for adding this test. Should we also add "14" and "17" here, or are we only generally testing Java 8 and Java 11?

Reply:
The main change that is relevant here divides JDK 8 and everything that comes after, and 11 just happens to be the most common one :shrug: